yarvt

```diff
@@ -540,17 +540,28 @@ function build_rootfs () {
 }
 
 function bootstrap () {
-	build_toolchain newlib
-	build_toolchain glibc
-	build_toolchain musl32
-	build_toolchain musl64
-	build_qemu
+        # if one step fails then "bootstrap" fails
+        if  build_toolchain newlib  &&
```

Comment:
If they all return 0, then you return -1, this doesn't make much sense.

Reply:
`return 0` means that the function was successful, and as result the if statement will succeed, please test it, it works fine. On the other hand, if for example `build_toolchain glibc` fails then the `if` statement fails, -1 is returned and logs are not deleted because of the changes below.

Reply:
Argh I forgot that for bash 0 is true when it comes from a function, I'm used to checking this explicitly (if [[ $? == 0 ]]) but your way also works as expected.